proposals/lip-0022.md

```diff
@@ -149,7 +149,7 @@ A round is composed of 103 blocks now.
 
 Before being ordered according to [LIP 0003](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0003.md) to generate the forging list, the set of forging delegates for `round` + 1 is constructed as follows:
 
-* The addresses of the top 101 delegates by delegate weight at the end of `round` - 2 are added to the set (the 2 rounds delay is specified in [Change of Delegates](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0014.md#change-of-delegates) subsection of LIP 0014). Ties are broken in favor of the smaller address.
+* The addresses of the top 101 delegates by delegate weight at the end of `round` - 2 are added to the set (the 2 rounds delay is specified in [Change of Delegates](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0014.md#change-of-delegates) subsection of LIP 0014). Ties are broken in favor of the smaller address. In the case that 101 or less delegates are available, the addresses of the available delegates are selected and the round computation ends here.
```

Comment:
I would prefer to use an enumerated list here because these are actually two steps that have to be done one after the other.